### ISSUE TITLE
Add rpm package for Fedora 38

### ISSUE THREAD

**Issue by dbronecki** [docker/docker-ce-packaging#881]
There is no docker-ce and docker-compose-plugin under Fedora 38 packages (https://download.docker.com/linux/fedora/38/x86_64/stable/Packages/). Only containerd.io is available. Fedora 38 was released yesterday, so it would be nice to have it

**Comment by thaJeztah** [comment]
Yeah, I think tini is broken on Fedora 38; see https://github.com/moby/moby/issues/44808

We contributed a fix https://github.com/krallin/tini/pull/206 but don't think it was released yet

**Comment by thaJeztah** [comment]
(not 100% sure now it that fix was related - typing from my phone ☺️😅)

**Comment by Romain-Geissler-1A** [comment]
Do you want me to have a look ? I think we should also add fedora 39 now (to avoid the same issue in 6 months ;) )

**Comment by thaJeztah** [comment]
Yes, feel free to give it a try to see if an updated version builds (if it does, we should try to have a new release tagged for tini)

**Comment by thaJeztah** [comment]
Oh, thanks! Was there anything you changed to make it work? (curious now)

For fedora 39, I think that one is still same as  "rawhide" (I know we have an entry for that in containerd packaging, and usually start building / publishing packages when it gets closer to a release)

**Comment by Romain-Geissler-1A** [comment]
No change at all. I copied the Dockerfile of fedora 37, replaced 37 by 38 inside the file, and it worked (your Jenkins CI doesn't trust Jenkinsfile changes, so I had to test locally, and it worked for me locally on an aarch64 machine).

So for fedora 39 we just do nothing for now ? Or keeping both 39 and rawhide is ok ? https://github.com/docker/containerd-packaging/pull/315/files

**Comment by XaiMeS** [comment]
Packages from Fedora 37 work fine

**Comment by thaJeztah** [comment]
Fixed by https://github.com/docker/docker-ce-packaging/pull/886 (once we do a new patch release / deploy)
